### ISSUE TITLE
Wattage randomly lowering

### ISSUE THREAD

**Issue by chestm007** [MicroSur/sur_myevic#40]
**Box Model:** wismec releaux rx-gen3

**Coil setup:** dual claptons (0.18 ohm) in a geekvape zeus dual rta

First of all thankyou for all your hard work with this software!!

I'm not entirely sure if this is a hardware issue for me specifically or potentially an extremely strange bug, so i figured i'd raise an issue and see if anyone else is facing this also

Occasionally my wattage changes dramatically for seemingly no reason. I normally have my current setup at 110w but sometimes my mod will set itself to anything from 30-80watts.
This has been happening for roughly the last 4 months.

Thankyou again!!

**Comment by arafey** [comment]
It could be the PwrLo feature. From [the documentation](https://github.com/MicroSur/sur_myevic/blob/master/git_doc_ru/vaping_ru.md) (translated to English):

> PwrLo (power down) If on, the mod will automatically lower the installed power if there is not enough battery power for this.

Try disabling it and see what happens.

**Comment by MicroSur** [comment]
chestm007
I need your settings file, dataflash. Use Save DFlash button in FWUpdater.
Do you use auto-detect profiles? Erase unused or if their resistances are similar.

**Comment by chestm007** [comment]
@arafey I dont use that feature, so its already disabled - thanks though

@MicroSur 
[gen3-data.zip](https://github.com/MicroSur/sur_myevic/files/2971916/gen3-data.zip)

**Comment by MicroSur** [comment]
I have to do research in the firmware for myself, why power can fall generally. 
If one coil falls off on vaping, for example.

But in your settings I see a few errors. Set the correct version of the hardware in the FWUpdater, to 1.00 and Reset DFlash. Use the latest available SME-firmware.

**Comment by chestm007** [comment]
@MicroSur i cant figure out how to set the firmware version anywhere - any pointers would be amazing :)

i tried setting the keylock after putting watts at 110, and it still changed itself, so it looks like a software/deep hardware issue.

also, i reset the dataflash anyway and it seems the problem hasnt occured since - perhaps my mod was doing wierd stuff based on bad flash memory.... who knows at this stage :/

**Comment by MicroSur** [comment]
@chestm007 Hardware version, FWUpdater, Set HW version button.

**Comment by MicroSur** [comment]
If the HW version is not correct, it means that the rest of the settings could have been corrupted. Therefore, I suggested a reset.

**Comment by chestm007** [comment]
@MicroSur ok, I've set the hardware version to 1.00, and installed the latest firmware - the issue was still occuring... 

this next part makes me feel like an idiot -- i noticed shunt resistance was spiking to ~0.5 ohms after chain vaping a few times, this is normally where the wattage reduction occurs, i also noticed the bottom of my vape was getting hot, concentrated in one area... turns out one of the screws holding the battery door on was loose, i've tightened it and now SHR is a steady 0.2 ohms (where it was returning to before after not vaping on it for a bit) - its been an hour so far without an issue, which is longer then ever before, i cautiously think the issue is resolved

TL;DR I think this was a hardware issue, I'm an idiot, sorry for wasting your time :/

:heart: thanks for looking into this for me though, i greatly appreciate your help! once again - thank you for this amazing firmware!!!

**Comment by MicroSur** [comment]
Thank you too. 
Bad contacts are a problem for many Chinese mods.
In the info-line, select show real resistance and look at it, better than at SHR.

**Comment by chestm007** [comment]
Thanks for the info!

Going to close this issue since i can confirm this hasn't happened since tightening the screw :man_facepalming: 

Thanks again!!
